src/Common/APICollection.php

```diff
@@ -88,6 +88,16 @@ public function links($presenter)
     }
 
     /**
+     * Get the first item from the collection.
+     *
+     * @return APIResponse|null
+     */
+    public function first()
+    {
+        return $this->count() > 0 ? $this->items[0] : null;
```

Comment:
Yeah, I always go back and forth on that. Definitely not necessary, but I feel like it adds to readability in this scenario (since it seems weird to say something "has a count")?

No strong feeling either way though.